src/Lunr/Ticks/PHPMailer/PHPMailer.php

```diff
@@ -0,0 +1,219 @@
+<?php
+
+/**
+ * This file contains the PHPMailer class.
+ *
+ * SPDX-FileCopyrightText: Copyright 2025 Framna Netherland B.V., Zwolle, The Netherlands
+ * SPDX-License-Identifier: MIT
+ */
+
+namespace Lunr\Ticks\PHPMailer;
+
+use Lunr\Ticks\AnalyticsDetailLevel;
+use Lunr\Ticks\EventLogging\EventLoggerInterface;
+use Lunr\Ticks\TracingControllerInterface;
+use Lunr\Ticks\TracingInfoInterface;
+use PHPMailer\PHPMailer\PHPMailer as BaseMailer;
+
+/**
+ * PHPMailer class
+ */
+class PHPMailer extends BaseMailer
+{
+
+    /**
+     * Shared instance of the event logger
+     * @var EventLoggerInterface
+     */
+    protected readonly EventLoggerInterface $eventLogger;
+
+    /**
+     * Shared instance of the info tracing controller
+     * @var TracingControllerInterface&TracingInfoInterface
+     */
+    protected readonly TracingControllerInterface&TracingInfoInterface $tracingController;
+
+    /**
+     * Profiling level
+     * @var AnalyticsDetailLevel
+     */
+    protected AnalyticsDetailLevel $analyticsDetailLevel;
+
+    /**
+     * Start time of the mail sending
+     * @var float
+     */
+    protected float $startTimestamp;
+
+    /**
+     * Constructor.
+     *
+     * @param bool|null $exceptions Should we throw external exceptions?
+     */
+    public function __construct(?bool $exceptions = NULL)
+    {
+        $this->analyticsDetailLevel = AnalyticsDetailLevel::None;
+
+        parent::__construct($exceptions);
+
+        /**
+         * The type of action_function is defined as string but it should be callable, so we ignore the phpstan warning
+         * @phpstan-ignore assign.propertyType
+         */
+        $this->action_function = [ $this, 'afterSending' ]; // phpcs:ignore Lunr.NamingConventions.CamelCapsVariableName
```

Comment:
This only catches successful sends. Failures would throw Exceptions and exit `send()` before `doCallback()` is invoked